packages/@react-stately/table/src/useTableColumnResizeState.ts

```diff
@@ -64,6 +64,7 @@ export function useTableColumnResizeState<T>(props: TableColumnResizeStateProps<
     getDefaultMinWidth,
     onColumnResizeStart: propsOnColumnResizeStart,
     onColumnResizeEnd: propsOnColumnResizeEnd,
+    onColumnResize: propsOnColumnResize,
```

Comment:
I'm not entirely convinced we need this tbh, doesn't really seem to differ from `onResize` from useTableColumnResize

Reply:
Are we actually using `onColumnResizeStart` and `onColumnResizeEnd`? I think those two should go away in order to match TableLayout
https://github.com/adobe/react-spectrum/blob/main/packages/%40react-stately/layout/src/TableLayout.ts#L94

Reply:
Technically unnecessary for the aria example, but `onColumnResizeStart` and `onColumnResizeEnd` provide different info from their `onResize` counterparts, namely they return the resize column key.

Reply:
Yes, but the key is only used by the stately/layout, I don't think anyone needs to have that, they already have the info from the stately object https://github.com/adobe/react-spectrum/blob/f7693f1cd7e0e44ab113d15e42d0777e1aa9c5e3/packages/%40react-stately/table/src/useTableColumnResizeState.ts#L119